core/src/plugins/filed/python/module/bareosfd.cc

```diff
@@ -885,8 +885,11 @@ static inline bool PyAclPacketToNative(PyAclPacket* pAclPacket,
     }
 
     if (ap->content) { free(ap->content); }
-    ap->content = (char*)malloc(ap->content_length);
-    memcpy(ap->content, buf, ap->content_length);
+    ap->content = (char*)malloc(ap->content_length + 1);
+    memcpy(ap->content, buf, ap->content_length + 1);
+  } else {
+    PyErr_SetString(PyExc_TypeError,
+                    "acl packet content needs to be of bytearray type");
```

Comment:
it is correct to fail still return true here?

Reply:
Good catch!